### PR TITLE
misc: rename library to solar_client

### PR DESCRIPTION
**Pull request by c0nsol3**

### CODE REVIEW

**Comment by sleepdefic1t** [comment]
@c0nsol3 
Can you revert the formatting changes in `solar_client/api/delegates.py`, please and thank you 🙏 

It's only one file but the application of formatting should be a separate thing.

**Comment by c0nsol3** [comment]
@sleepdefic1t done

File: README.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
 > A simple Python API client for the Solar Blockchain.
 
+## Install
+
+`pip install solar-client`
+
+## Example usage
+
+```py
+from client.client import SolarClient
```
Comment (sleepdefic1t) [inline README.md:12]: Should this be `solar_client`?
```diff
- from client.client import SolarClient
+ from solar_client.client import SolarClient
```

**Review by sleepdefic1t — CHANGES_REQUESTED** [review]

**Comment by c0nsol3** [comment]
@sleepdefic1t done

**Review by sleepdefic1t — APPROVED** [review]

**Review by Cactus-15-49 — APPROVED** [review]

**Pull request merged**